lms/djangoapps/verify_student/toggles.py

```diff
@@ -0,0 +1,30 @@
+"""
+Toggles for verify_student app
+"""
+
+from openedx.core.djangoapps.waffle_utils import WaffleFlagNamespace, WaffleFlag
+
+# Namespace for verify_students waffle flags.
+WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(name='verify_student')
+
+# Waffle flag to use new email templates for sending ID verification emails.
+# .. toggle_name: verify_student.use_new_email_templates
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Supports staged rollout to students for a new email templates implementation for ID verification.
+# .. toggle_category: verify student
+# .. toggle_use_cases: incremental_release, open_edx
+# .. toggle_creation_date: 2020-06-18
+# .. toggle_expiration_date: 2020-07-18
```

Comment:
Will this be the exact date? Because if we are not sure on the clear date, we can mark it optional.